src/content/configuration/optimization.md

```diff
@@ -236,3 +236,20 @@ module.exports = {
   }
 };
 ```
+
+## `optimization.flagIncludedChunks`
```

Comment:
What is its default value?

Reply:
unfortunately default is dynamic due to mode, if mode is production it will default to true as i mentioned.